custom/icds_reports/static/js/directives/kpi/kpi.directive.js

```diff
@@ -24,6 +24,10 @@ function KpiController($location, haveAccessToFeatures) {
     };
 
     this.isNumber = window.angular.isNumber;
+
+    this.toShowInKpi = function (cellLabel) {
```

Comment:
Could you add a comment that this is to be removed after QA?

Reply:
i beleive once verified, there will be one more ticket created to remove this from feature flag.Anyways i will add it